packages/TagPicker/src/TagPicker.tsx

```diff
@@ -283,6 +283,14 @@ const TagPicker: React.FunctionComponent<TagPickerProps> = ({
         }
     }, [tagRemoved, inputDisabled, handleGainFocus, autoFocus]);
 
+    useEffect(() => {
```

Comment:
What is the reason of this useEffect? It feels like something that we could handle in an action handler instead of a use effect. 

Why is the check in handleResultSelection not enough?

Reply:
Since on load you could already have selected tags, this assures that the user does not try to pick a tag when the limit is already reached, see issue https://github.com/gsoft-inc/ov-igloo-ui/pull/540